packages/nextjs/src/config/wrappers/withSentryGetServerSideProps.ts

```diff
@@ -1,6 +1,7 @@
+import { hasTracingEnabled } from '@sentry/tracing';
 import { GetServerSideProps } from 'next';
 
-import { callDataFetcherTraced } from './wrapperUtils';
+import { callServerSideDataFetcherWithTracingInstrumentation, withErrorInstrumentation } from './wrapperUtils';
```

Comment:
> callServerSideDataFetcherWithTracingInstrumentation

LOL. Even for my tastes this feels a little long (and that's saying something!). Unless this has a counterpart without tracing, maybe just knock the `WithTracingInstrumentation` off? Or maybe just `traceServerSideDataFetcher`? Or, if we're really on a `with` kick, `withServerSideTracing`? WDYT?

Reply:
Ok 🤣 changed in 93ebc28c9c4367b4ee307693f63cdcc7498f0838